messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/MessagingTransformerRegistration.java

```diff
@@ -83,10 +84,27 @@ public void registerTransformers(SubsystemTransformerRegistration registration)
         registerTransformers_EAP_7_1_0(builder.createBuilder(MessagingExtension.VERSION_3_0_0, MessagingExtension.VERSION_2_0_0));
         registerTransformers_EAP_7_0_0(builder.createBuilder(MessagingExtension.VERSION_2_0_0, MessagingExtension.VERSION_1_0_0));
 
-        builder.buildAndRegister(registration, new ModelVersion[] { MessagingExtension.VERSION_1_0_0, MessagingExtension.VERSION_2_0_0,
+        builder.buildAndRegister(registration, new ModelVersion[]{MessagingExtension.VERSION_1_0_0, MessagingExtension.VERSION_2_0_0,
             MessagingExtension.VERSION_3_0_0, MessagingExtension.VERSION_4_0_0, MessagingExtension.VERSION_5_0_0,
             MessagingExtension.VERSION_6_0_0, MessagingExtension.VERSION_7_0_0, MessagingExtension.VERSION_8_0_0,
-            MessagingExtension.VERSION_9_0_0, MessagingExtension.VERSION_10_0_0, MessagingExtension.VERSION_11_0_0, MessagingExtension.VERSION_12_0_0});
+            MessagingExtension.VERSION_9_0_0, MessagingExtension.VERSION_10_0_0, MessagingExtension.VERSION_11_0_0,
+            MessagingExtension.VERSION_12_0_0, MessagingExtension.VERSION_13_0_0});
+    }
+
+    private static void registerTransformers_WF_23(ResourceTransformationDescriptionBuilder subsystem) {
+        ResourceTransformationDescriptionBuilder server = subsystem.addChildResource(MessagingExtension.SERVER_PATH);
+        rejectDefinedAttributeWithDefaultValue(server,
+                ServerDefinition.NETWORK_CHECK_LIST,
+                ServerDefinition.NETWORK_CHECK_NIC,
+                ServerDefinition.NETWORK_CHECK_PERIOD,
+                ServerDefinition.NETWORK_CHECK_PING6_COMMAND,
+                ServerDefinition.NETWORK_CHECK_PING_COMMAND,
+                ServerDefinition.NETWORK_CHECK_TIMEOUT,
+                ServerDefinition.NETWORK_CHECK_URL_LIST,
```

Comment:
These NETWORK_CHECK attributes were all in WF 22 and are handled in registerTransformers_WF_22.